internal/container/docker.go

```diff
@@ -229,7 +229,7 @@ func (e *dockerImpl) ContainerList(ctx context.Context, names ...string) ([]Cont
 
 	if len(names) > 0 {
 		for _, name := range names {
```

Comment:
```suggestion
		for _, label := range labels {
```
let's rename `names` to `labels` to match the filter